### ISSUE TITLE
Results of Accessibility Testing May 19, 2024 (Dev Preview 0.3.0)

### ISSUE THREAD

**Issue by kevinandrews1** [bbertucc/equalify#339]
This stems from the recent chat on the a11y channel of our Slack (cc @wilsuriel03 & @heythisischris. 

## Stats
- Windows 11,
- Google Chrome Version 125.0.6422.61
- JAWS 2024.2312.53.400,
- NVDA 2024.1
- Equalify.dev 0.3.0 Developer Preview

If any of the issues are particular to a browser, I make that clear, though that wasn't the case here.

### Signup/signin
- Signin page uses placeholder for the email. you shouldn't use placeholder text here --that really goes for any input field. These tend to introduce color contrast issues. Also the placeholder will disappear as soon as text is entered, so you should consider wrapping it in a p tag and associate it with the input using aria-describedby.
- On the signup page, https://equalify.dev/signup, the first field is read as "email first name". Next field appears to read helper text as the label for the first name only rather than explicitly saying "first name".
- When attempting to sign up, after pressing Continue, my focus jumped back to the confirm password field but there was no context around the issue. It needs to communicate to the user that the passwords don't match, in this case. In other UIs it might be that the password wasn't strong enough, didn't contain the required character types, etc. Communicating the issue eliminates any of the ambiguity. You could consider comparing the two password fields before the user presses Continue if then once they've entered into the confirm field, wait a beat and confirm whether or not the passwords match so they don't have to go backward through the DOM and guess what the issue was, creating additional work for them and disrupting the workflow.
- After pressing Continue, I did receive the verification code, but there seemed to be stale content from earlier when I tried signing in with my email. If it confused me, it will most certainly confuse other users.

### Main screen after logging in/Reports
- Navigation in the main navigation landmark label which is redundant. It reads as "main navigation navigation".
- View Report link is redundant, especially if the user navigates via the tab key or pulls up a links list, which is a common feature of most modern screen readers. It lacks context pertaining to which report is being viewed. E.g. change the link text to "View Accessibility 1 Report". Make sure the accessible name and visible label match if at all possible. This is important for two reasons:
    - Speech-input users can directly activate controls on a page with fewer surprising changes of focus, and
    - text-to-speech users will have a better experience because the labels that they hear match the visible text labels that they see on the screen.
- Viewing a report, in this case Accessibility Audit 1, there are a variety of graphics before the message table with generic-sounding alt text whose purpose is unclear: "2021/01", "2021/02", "2021/03", "2021/04", "2021/05", "0", "8", "16", "24", and "32"".
- Report/edit/danger zone: When closing/cancelling deletion, screen reader focus is unclear and focus should return to the button that initiated the dialog in this case being the delete button.
- Searching any of the selected filter types does not announce the search result even if no items are found.
- When creating a report, the report name text field is unavailable.

This random one I didn't know how to categorize: When going to Properties I receive the messsage "Error loading properties".
